tests/app/dao/notification_dao/test_notification_dao.py

```diff
@@ -799,20 +799,6 @@ def test_should_delete_only_notification_with_id(
     assert notify_db_session.session.get(Notification, to_delete_notification.id) is None
 
 
-@pytest.mark.serial
-def test_should_delete_no_notifications_if_no_matching_ids(
```

Comment:
This test was pointless. It made sure nothing was deleted but the calling code just did a get on the primary key, so of course if it doesn't find the PK there won't be a delete.